common/src/API.php

```diff
@@ -63,6 +63,26 @@ private function buildURL($address, $parameters) {
         }
         return $address . '&' . $query_string;
     }
+
+    public function hasConsent(array $data): bool {
+        $params = [
+            'order_number' => $data['order'],
+            'id' => $this->shop_id,
+            'code' => $this->api_key,
+        ];
+        $permission_url = $this->buildURL('https://' . $this->api_domain . '/api/2.0/order_permissions.json', $params);
+        $permission_response = Requests::get($permission_url, []);
```

Comment:
```suggestion
        $permission_response = Requests::get($permission_url);
```

No need for the second parameter? https://developer.wordpress.org/reference/classes/requests/get/

Reply:
I think this is a deprecated library, could you also check that?